lib/config/executor/container_vm.go

```diff
@@ -44,6 +44,21 @@ type Common struct {
 	Notes string `vic:"0.1" scope:"hidden" key:"notes"`
 }
 
+// Common data (specifically for a containerVM) between managed entities, across execution environments.
+type CommonSpecForVM struct {
```

Comment:
@gigawhitlocks See the scope of name `Name string `vic:"0.1" scope:"hidden" key:"name"``